edx_proctoring/backends/backend.py

```diff
@@ -18,6 +18,8 @@ class ProctoringBackendProvider(six.with_metaclass(abc.ABCMeta)):
     tech_support_phone = ''
     # whether this backend supports an instructor review/configuration dashboard
     has_dashboard = False
+    # whether practice exams map to "onboarding" exams for this backend
+    supports_onboarding = False
```

Comment:
Do you need to add a corresponding property to `rest.py` of `supports_onboarding = True`?

Reply:
It can go there, or on the subclass of the rest backend. But you're probably right that it'd be expedient to add it to the rest base class.

Reply:
If it is to go on the subclass, we should update the docs to indicate that.

Reply:
I just added it to the rest backend.